lib/chef/provider/chef_group.rb

```diff
@@ -15,13 +15,13 @@ def whyrun_supported?
       if differences.size > 0
         description = [ "update group #{new_resource.name} at #{rest.url}" ] + differences
         converge_by description do
-          rest.put("groups/#{new_resource.name}", normalize_for_put(new_json))
+          rest.put("organizations/#{new_resource.org}/groups/#{new_resource.name}", normalize_for_put(new_json))
```

Comment:
Yes, the convention is that :chef_server_url is always the thing just above /nodes, /clients, etc.  The reason not to change it at the moment is that this variable corresponds to the [Chef::Config.chef_server_url](https://docs.chef.io/config_rb_knife.html#settings) variable, which refers to that with such ubiquity that changing it would be rather issueful.  There is a new config variable coming down the pike that refers to the root, but until that lands what I've been doing is assuming rest refers to "organizations/blah" and using `rest.root_url` instead of a relative PUT: https://github.com/chef/cheffish/blob/master/lib/chef/provider/chef_organization.rb#L18

That makes it so you can respect users who leave off /organizations/orgname, and support users who put it on, all with the same code.